src/main/java/com/github/onsdigital/babbage/configuration/Configuration.java

```diff
@@ -122,6 +126,7 @@ public static String getDefaultContentDatePattern() {
         private static String elasticSearchIndexAlias = defaultIfBlank(getValue("ELASTIC_SEARCH_INDEX_ALIAS"), "ons");
         private static Integer elasticSearchPort = Integer.parseInt(defaultIfBlank(getValue("ELASTIC_SEARCH_PORT"), "9300"));
         private static String elasticSearchCluster = defaultIfBlank(getValue("ELASTIC_SEARCH_CLUSTER"), "");
+        private static final String HIGHLIGHT_URL_BLACKLIST_FILE = "highlight-url-blacklist";
```

Comment:
Get this from an env var and use a default if empty - follow the pattern for the existing config stuff